develop/toc.yml

```diff
@@ -669,6 +669,8 @@ items:
           topicUid: System_FormatException
         - name: System.Collections.Generic.KeyNotFoundException
           topicUid: System_Collections_Generic_KeyNotFoundException
+        - name: Exception in Finalizer
```

Comment:
Added to TOC but [QAction_exceptions](https://github.com/SkylineCommunications/dataminer-docs/pull/3644/files#diff-65dc9ba3247129b234f8d3ca76acc8c5d3c6fdb6b17114e18432c4502f09e3fbR662) is also a page with a list that could probably be adjusted.

Reply:
updated